### ISSUE TITLE
Bug: Container fails to download Unbound Files

### ISSUE THREAD

**Issue by Kieffer87** [qdm12/private-internet-access-docker#152]
**TLDR**: After 754bab9763e0d757794d1406931e44d3fa412073 commit was merged, container fails to download unbound files. Disabling DOT allows the container to starts as usual.

1. Is this urgent?

    - [x] Yes (Breaks DOT)
    - [] No

2. What VPN service provider are you using?

    - [x] PIA
    - [ ] Mullvad
    - [ ] Windscribe

3. What's the version of the program?
    `Running version latest built on 2020-05-02T17:09:01Z (commit 754bab9)`

4. What are you using to run the container?

    - [ ] Docker run
    - [x] Docker Compose
    - [ ] Kubernetes
    - [ ] Docker stack
    - [ ] Docker swarm
    - [ ] Podman
    - [ ] Other:

5. Extra information

Logs:

```log
ErrorWarningSystemArrayLogin


=========================================
============= PIA container =============
========== An exquisite mix of ==========
==== OpenVPN, Unbound, DNS over TLS, ====
===== Shadowsocks, Tinyproxy and Go =====
=========================================
=== Made with ❤️ by github.com/qdm12 ====
=========================================

Running version latest built on 2020-05-02T17:09:01Z (commit 754bab9)


📣 New HTTP control server, see https://github.com/qdm12/private-internet-access-docker#http-control-server

🔧 Need help? https://github.com/qdm12/private-internet-access-docker/issues/new
💻 Email? quentin.mcgaw@gmail.com
☕ Slack? Join from the Slack button on Github
💸 Help me? https://github.com/sponsors/qdm12
2020-05-03T11:23:12.834-0500 INFO OpenVPN version: 2.4.8

2020-05-03T11:23:12.869-0500 INFO Unbound version: 1.9.6

2020-05-03T11:23:12.887-0500 INFO IPtables version: v1.8.3

2020-05-03T11:23:12.913-0500 INFO TinyProxy version: 1.10.0

2020-05-03T11:23:12.979-0500 INFO ShadowSocks version: 3.3.4

2020-05-03T11:23:12.979-0500 INFO Settings summary below:
OpenVPN settings:
|--Network protocol: udp
|--Verbosity level: 1
|--Run as root: yes
|--Target IP address: <nil>
|--Custom cipher: aes-256-gcm
|--Custom auth algorithm: sha256
PIA settings:
|--User: [redacted]
|--Password: [redacted]
|--Region: us texas
|--Encryption: strong
|--Port forwarding: off
System settings:
|--User ID: 1000
|--Group ID: 1000
|--Timezone: america/chicago
|--IP Status filepath: /ip
DNS over TLS settings:
|--DNS over TLS provider:
|--cloudflare
|--Caching: enabled
|--Block malicious: enabled
|--Block surveillance: enabled
|--Block ads: disabled
|--Allowed hostnames:
|--
|--Private addresses:
|--127.0.0.1/8
|--10.0.0.0/8
|--172.16.0.0/12
|--192.168.0.0/16
|--169.254.0.0/16
|--::1/128
|--fc00::/7
|--fe80::/10
|--::ffff:0:0/96
|--Verbosity level: 1/5
|--Verbosity details level: 0/4
|--Validation log level: 0/2
|--IPv6 resolution: enabled
Firewall settings:
|--Allowed subnets: 192.168.0.0/16
TinyProxy settings: disabled
ShadowSocks settings: disabled

2020-05-03T11:23:12.980-0500 INFO openvpn configurator: checking for device /dev/net/tun
2020-05-03T11:23:12.980-0500 WARN TUN device is not available: open /dev/net/tun: no such file or directory
2020-05-03T11:23:12.980-0500 INFO openvpn configurator: creating /dev/net/tun
2020-05-03T11:23:12.980-0500 INFO openvpn configurator: writing auth file /etc/openvpn/auth.conf
2020-05-03T11:23:12.980-0500 INFO routing: detecting default network route
2020-05-03T11:23:12.981-0500 INFO routing: default route found: interface eth0, gateway 192.168.1.1, subnet 192.168.1.0/24
2020-05-03T11:23:12.981-0500 INFO firewall configurator: accepting all traffic
2020-05-03T11:23:12.985-0500 INFO Launching standard output merger
2020-05-03T11:23:12.985-0500 INFO routing: adding 192.168.0.0/16 as route via eth0
2020-05-03T11:23:12.988-0500 INFO firewall configurator: clearing all rules
2020-05-03T11:23:12.993-0500 INFO firewall configurator: blocking all traffic
2020-05-03T11:23:12.999-0500 INFO firewall configurator: creating general rules
2020-05-03T11:23:13.009-0500 INFO firewall configurator: allowing output traffic to VPN server 162.216.46.15 through eth0 on port udp 1197
2020-05-03T11:23:13.011-0500 INFO firewall configurator: allowing output traffic to VPN server 162.216.46.30 through eth0 on port udp 1197
2020-05-03T11:23:13.012-0500 INFO firewall configurator: allowing output traffic to VPN server 162.216.46.48 through eth0 on port udp 1197
2020-05-03T11:23:13.015-0500 INFO firewall configurator: allowing output traffic to VPN server 162.216.46.55 through eth0 on port udp 1197
2020-05-03T11:23:13.016-0500 INFO firewall configurator: allowing output traffic to VPN server 162.216.46.68 through eth0 on port udp 1197
2020-05-03T11:23:13.018-0500 INFO firewall configurator: allowing output traffic to VPN server 162.216.46.95 through eth0 on port udp 1197
2020-05-03T11:23:13.020-0500 INFO firewall configurator: allowing output traffic to VPN server 162.216.46.116 through eth0 on port udp 1197
2020-05-03T11:23:13.022-0500 INFO firewall configurator: allowing output traffic to VPN server 162.216.46.142 through eth0 on port udp 1197
2020-05-03T11:23:13.023-0500 INFO firewall configurator: allowing output traffic to VPN server 162.216.46.166 through eth0 on port udp 1197
2020-05-03T11:23:13.025-0500 INFO firewall configurator: allowing output traffic to VPN server 162.216.46.174 through eth0 on port udp 1197
2020-05-03T11:23:13.029-0500 INFO firewall configurator: accepting input and output traffic for 192.168.1.0/24
2020-05-03T11:23:13.032-0500 INFO firewall configurator: accepting input traffic through eth0 from 192.168.0.0/16 to 192.168.1.0/24
2020-05-03T11:23:13.033-0500 INFO firewall configurator: accepting output traffic through eth0 from 192.168.1.0/24 to 192.168.0.0/16
2020-05-03T11:23:13.035-0500 WARN http server: restartOpenvpn function is not set, waiting...
2020-05-03T11:23:13.035-0500 INFO openvpn: starting
2020-05-03T11:23:13.035-0500 INFO openvpn configurator: starting openvpn
2020-05-03T11:23:13.036-0500 WARN http server: restartUnbound function is not set, waiting...
2020-05-03T11:23:13.038-0500 INFO openvpn: Sun May 3 11:23:13 2020 OpenVPN 2.4.8 x86_64-alpine-linux-musl [SSL (OpenSSL)] [LZO] [LZ4] [EPOLL] [MH/PKTINFO] [AEAD] built on Feb 7 2020
2020-05-03T11:23:13.038-0500 INFO openvpn: Sun May 3 11:23:13 2020 library versions: OpenSSL 1.1.1g 21 Apr 2020, LZO 2.10
2020-05-03T11:23:13.043-0500 INFO openvpn: Sun May 3 11:23:13 2020 TCP/UDP: Preserving recently used remote address: [AF_INET]162.216.46.55:1197
2020-05-03T11:23:13.043-0500 INFO openvpn: Sun May 3 11:23:13 2020 UDP link local: (not bound)
2020-05-03T11:23:13.043-0500 INFO openvpn: Sun May 3 11:23:13 2020 UDP link remote: [AF_INET]162.216.46.55:1197
2020-05-03T11:23:13.419-0500 INFO openvpn: Sun May 3 11:23:13 2020 WARNING: 'link-mtu' is used inconsistently, local='link-mtu 1550', remote='link-mtu 1542'

2020-05-03T11:23:13.420-0500 INFO openvpn: Sun May 3 11:23:13 2020 WARNING: 'cipher' is used inconsistently, local='cipher AES-256-GCM', remote='cipher BF-CBC'

2020-05-03T11:23:13.420-0500 INFO openvpn: Sun May 3 11:23:13 2020 WARNING: 'auth' is used inconsistently, local='auth [null-digest]', remote='auth SHA1'

2020-05-03T11:23:13.420-0500 INFO openvpn: Sun May 3 11:23:13 2020 WARNING: 'keysize' is used inconsistently, local='keysize 256', remote='keysize 128'

2020-05-03T11:23:13.420-0500 INFO openvpn: Sun May 3 11:23:13 2020 [aeac0f0952e14dc9a79259ca5768dc0e] Peer Connection Initiated with [AF_INET]162.216.46.55:1197
2020-05-03T11:23:14.698-0500 INFO openvpn: Sun May 3 11:23:14 2020 TUN/TAP device tun0 opened
2020-05-03T11:23:14.698-0500 INFO openvpn: Sun May 3 11:23:14 2020 /sbin/ip link set dev tun0 up mtu 1500
2020-05-03T11:23:14.699-0500 INFO openvpn: Sun May 3 11:23:14 2020 /sbin/ip addr add dev tun0 local 10.77.10.6 peer 10.77.10.5
2020-05-03T11:23:14.703-0500 INFO openvpn: Sun May 3 11:23:14 2020 Initialization Sequence Completed
2020-05-03T11:23:14.703-0500 INFO dns configurator: using DNS address 2606:4700:4700::1111 internally
2020-05-03T11:23:14.703-0500 INFO dns configurator: using DNS address 2606:4700:4700::1111 system wide
2020-05-03T11:23:14.703-0500 INFO Tunnel IP is 162.216.46.55, see more information at https://ipinfo.io/162.216.46.55
2020-05-03T11:23:14.704-0500 INFO dns configurator: downloading root hints from https://raw.githubusercontent.com/qdm12/files/master/named.root.updated
2020-05-03T11:23:14.705-0500 ERROR unbound dns over tls setup: cannot GET content of URL https://raw.githubusercontent.com/qdm12/files/master/named.root.updated: Get "https://raw.githubusercontent.com/qdm12/files/master/named.root.updated": dial tcp: lookup raw.githubusercontent.com on [2606:4700:4700::1111]:53: dial udp [2606:4700:4700::1111]:53: connect: cannot assign requested address
```

Configuration file:

```yml
OpenVPN settings:
|--Network protocol: udp
|--Verbosity level: 1
|--Run as root: yes
|--Target IP address: <nil>
|--Custom cipher: aes-256-gcm
|--Custom auth algorithm: sha256
PIA settings:
|--User: [redacted]
|--Password: [redacted]
|--Region: us texas
|--Encryption: strong
|--Port forwarding: off
System settings:
|--User ID: 1000
|--Group ID: 1000
|--Timezone: america/chicago
|--IP Status filepath: /ip
DNS over TLS settings:
|--DNS over TLS provider:
|--cloudflare
|--Caching: enabled
|--Block malicious: enabled
|--Block surveillance: enabled
|--Block ads: disabled
|--Allowed hostnames:
|--
|--Private addresses:
|--127.0.0.1/8
|--10.0.0.0/8
|--172.16.0.0/12
|--192.168.0.0/16
|--169.254.0.0/16
|--::1/128
|--fc00::/7
|--fe80::/10
|--::ffff:0:0/96
|--Verbosity level: 1/5
|--Verbosity details level: 0/4
|--Validation log level: 0/2
|--IPv6 resolution: enabled
Firewall settings:
|--Allowed subnets: 192.168.0.0/16
TinyProxy settings: disabled
ShadowSocks settings: disabled
```

Host OS:
Unraid

**Comment by qdm12** [comment]
Because you turned on DOT_IPV6 it tries to reach the ipv6 dns address. I think maybe pia doesn't support ipv6 (or your host). Turning it off will pick the ipv4 dns address and will work. But please let me know if you have ipv6 enabled in your network or if ipv6 works with your VPN provider, I'd be curious!

**Comment by Kieffer87** [comment]
That did the trick, thanks for the quick response.
